contracts/LotteryNFT.sol

```diff
@@ -14,7 +14,7 @@ contract LotteryNFT is ERC721, Ownable {
     mapping (uint256 => uint256) public issueIndex;
     mapping (uint256 => bool) public claimInfo;
 
-    constructor() public ERC721("Pancake Lottery Ticket", "PLT") {}
+    constructor() public ERC1155("random meta uri") {}
```

Comment:
@ChefNyan - Maybe we should keep `"Pancake Lottery Ticket", "PLT"` here?

Reply:
Not actually. ERC1155 accepts uri in constructor

Reply:
`ERC1155("")` should work!

Reply:
Ok